Feat: make examples independent (#1334)
stoplightio/elements: README.md

- **[bootstrap](./examples/bootstrap)** - A single HTML page utilizing the Web Components distribution via a global script tag.

To run these examples yourself:
1. Clone this repo.

> 2. Go to `/examples` folder within a cloned repo and open a command line in a given example directory.

Reviewer: ```suggestion
2. Go to `examples` folder and open an example, e.g.: `examples/angular`.
```
